File: WordpressExtractorModularApp/WordpressExtractorModularApp/WordpressExtractorModular/Models/Comment.cs
using System;

namespace WordpressExtractor.Models
{
    public class Comment
    {
        public int CommentId { get; set; }
        public int PostId { get; set; }
        public string Author { get; set; } = string.Empty;
        public string AuthorEmail { get; set; } = string.Empty;
        public string AuthorUrl { get; set; } = string.Empty;
        public string AuthorIp { get; set; } = string.Empty;
        public DateTime? Date { get; set; }
        public DateTime? DateGmt { get; set; }
        public string Content { get; set; } = string.Empty;
        public string Approved { get; set; } = string.Empty;
        public string Type { get; set; } = string.Empty;
        public int Parent { get; set; }
        public int UserId { get; set; }

        public override string ToString()
        {
            return $"Comment by {Author} on {Date?.ToShortDateString()}";
        }
    }
}

File: react-wordpress-importer/src/core/domain/types/Comment.ts
// react-wordpress-importer/src/types/Comment.ts

export interface Comment {
    CommentId: number;
    PostId: number;
    Author: string;
    AuthorEmail: string;
    AuthorUrl: string;
    AuthorIp: string;
    Date: Date | null;
    DateGmt: Date | null;
    Content: string;
    Approved: string;
    Type: string;
    Parent: number;
    UserId: number;
    // Add other properties as needed
}
